lib/screen/welcome/welcome_screen.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:showcase_flutter_build_web_desktop/theme/color.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import '../../gen/assets.gen.dart';
+
+class WelcomeScreen extends StatefulWidget {
+  const WelcomeScreen({super.key});
+
+  @override
+  State<StatefulWidget> createState() {
+    return _WelcomScreenState();
+  }
+}
+
+class _WelcomScreenState extends State<WelcomeScreen> {
+  AppLocalizations get _localizations => AppLocalizations.of(context)!;
+
+  TextTheme get _textTheme => Theme.of(context).textTheme;
+
+  @override
+  Widget build(BuildContext context) {
+    var screenSize = MediaQuery.of(context).size;
+
+    return Scaffold(
+      body: Container(
+        alignment: Alignment.center,
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            Image.asset(
+              Assets.images.leafLogo.path,
+              width: screenSize.height / 6,
+              height: screenSize.height / 6,
```

Comment:
Regarding this [commrnt](https://github.com/nimblehq/showcase-flutter-build-web-desktop/pull/9/files#r1252524925) we can get rid of calculating the size.
```suggestion
              width: 183, // todo Add in Metrics
              height: 183,
```

Reply:
Same here